packages/@react-spectrum/menu/src/ActionMenu.tsx

```diff
@@ -0,0 +1,39 @@
+import {ActionButton} from '@react-spectrum/button';
+import {filterDOMProps} from '@react-aria/utils';
+import {FocusableRef} from '@react-types/shared';
+import {Menu} from './Menu';
+import {MenuTrigger} from './MenuTrigger';
+import More from '@spectrum-icons/workflow/More';
+import React from 'react';
+import {SpectrumActionMenuProps} from '@react-types/menu';
+
+function ActionMenu<T extends object>(props: SpectrumActionMenuProps<T>, ref: FocusableRef<HTMLButtonElement>) {
+  let buttonProps = filterDOMProps(props);
+  if (buttonProps['aria-label'] === undefined) {
+    buttonProps['aria-label'] = 'More actions';
+  }
+  
+  return (
+    <MenuTrigger 
+      align={props.align}
+      direction={props.direction}
+      shouldFlip={props.shouldFlip}>
+      <ActionButton 
```

Comment:
I think we'd still want to expose the usual suspects for press so that people can add instrumentation for analytics. @dannify @devongovett other opinions?

Reply:
I would not mind adding but the description in the ticket says "convenience". I know I diverted already by extending `CollectionBase` so I would not mind adding them

Reply:
lets leave off for now then, we can always add it later if people need it